zcash_primitives/src/transaction/tests.rs

```diff
@@ -236,7 +236,12 @@ fn zip_0244() {
     fn to_test_txdata(
         tv: &self::data::zip_0244::TestVector,
     ) -> (TransactionData<TestUnauthorized>, TxDigests<Blake2bHash>) {
-        let tx = Transaction::read(&tv.tx[..], BranchId::Nu5).unwrap();
+        let tx = Transaction::read(
+            &tv.tx[..],
+            #[cfg(not(zcash_unstable = "nu6"))] /* TODO nu7 */ BranchId::Nu5,
```

Comment:
new line after `*/` (all occurrences)

Reply:
`cargo fmt` pushes it back to the above line (so making it new line would make it fail the Rustfmt CI check).

Reply:
I confirmed that this is the case for all lines where the newline is not used.